after-cluster.sh

```diff
@@ -3,4 +3,17 @@ docker pull nginx:1.13 # used by config-serve
 
 # Add images here for them to be available at runtime
 # for example:
-# docker pull quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.9.0
\ No newline at end of file
+# docker pull quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.9.0
+
+kubectl config set-context --current --namespace=kube-system
```

Comment:
specific to your needs?

Reply:
Well the csi-snapshotting etc becomes more and more mainstream (became beta in 1.17), so I left it in since I thought it might make sense to have this available for kind?

Reply:
as per other comment
if it was in kind proper, it would be part of the other setup scripts, after-cluster is really an extension point rather than a place for upstream scripts.